src/main/scala/com/mesosphere/sbt/BuildPlugin.scala

```diff
@@ -104,7 +104,6 @@ object BuildPlugin extends AutoPlugin {
   /** This should be appended to (testOptions in IntegrationTest) */
   def itTestOptions(
     javaHomeValue: Option[File],
-    classpathPrefix: Seq[File],
```

Comment:
@cruhland @larioj I made a non-backwards compatible change. We don't need this anymore now that we are not overriding the default repository through a resource file.

I am going to address your comments and merge this. The cosmos and cosmos-enterprise build will be broken until the other PRs are merged. They should be green by Monday.